browser/src/Editor/SimpleNeovimEditor.tsx

```diff
@@ -0,0 +1,120 @@
+/**
+ * NeovimEditor.ts
+ *
+ * IEditor implementation for Neovim
+ */
+
+import * as React from "react"
+
+import { IncrementalDeltaRegionTracker } from "./../DeltaRegionTracker"
+import { NeovimInstance } from "./../neovim"
+import { CanvasRenderer, DOMRenderer, INeovimRenderer } from "./../Renderer"
+import { NeovimScreen } from "./../Screen"
+
+import * as Config from "./../Config"
+
+// import { PluginManager } from "./../Plugins/PluginManager"
+
+import { Keyboard } from "./../Input/Keyboard"
+import { IEditor } from "./Editor"
+
+import { NeovimRenderer } from "./NeovimRenderer"
+
+export class DummyPluginManager {
+    public getAllRuntimePaths(): string[] {
+        return []
+    }
+
+    public startPlugins(neovimInstance: NeovimInstance): void {
+        // no-op
+    }
+}
+
+export class SimpleNeovimEditor implements IEditor {
+
+    private _neovimInstance: NeovimInstance
+    private _deltaRegionManager: IncrementalDeltaRegionTracker
+    private _renderer: INeovimRenderer
+    private _screen: NeovimScreen
+
+    private _pendingTimeout: any = null
+    private _pendingAnimationFrame: boolean = false
+
+    constructor(
+        private _config: Config.Config = Config.instance(),
+    ) {
+        // TODO: How to get rid of this?
+        this._neovimInstance = new NeovimInstance(new DummyPluginManager(), 100, 100)
```

Comment:
Oops, I meant the `DummyPluginManager`. It'd be nice to not need to send in this object at all - rethinking the relationship between NeovimInstance <-> PluginManager and revisiting that dependency.